app/logic/Deployments.scala

```diff
@@ -57,4 +64,17 @@ object Deployments {
       event.note,
       event.result.getOrElse(Succeeded)
     )
+
+  def buildLinks(deployment: Deployment, jiraResponse: Option[WSResponse]) = Kleisli[Future, JIRA.Context, Seq[Link]] {
```

Comment:
Maybe rename this to something like `enrichWithJiraInfo` and make it return an updated copy of the whole deployment, rather than just the links?